Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Start runtime image,
+FROM amsterdam/python:3.8-buster
+
+WORKDIR /app
+COPY . ./
+RUN pip install -e .
```

Comment:
Dit zou evt in een keer kunnen: `pip install -e .[django,tests]`

Reply:
Thnx. Done.